test/Microsoft.Buld.Sql.Tests/DotnetTestBase.cs

```diff
@@ -56,16 +61,42 @@ public void EnvironmentSetup()
             TestUtils.CopyDirectoryRecursive("../../../Template", this.WorkingDirectory);
 
             // Copy test specific files to WorkingDirectory
-            TestUtils.CopyDirectoryRecursive(this.TestDataDirectory, this.WorkingDirectory);
+            if (Directory.Exists(this.TestDataDirectory))
+            {
+                TestUtils.CopyDirectoryRecursive(this.TestDataDirectory, this.WorkingDirectory);
+            }
+        }
+
+        /// <summary>
+        /// Calls a dotnet command with <paramref name="dotnetCommandWithArgs"/>
+        /// </summary>
+        /// <returns>The Exit Code of the dotnet process</returns>
+        protected int RunGenericDotnetCommand(string dotnetCommandWithArgs, out string stdOutput, out string stdError)
+        {
+            // Set up the dotnet process
+            ProcessStartInfo dotnetStartInfo = new ProcessStartInfo
+            {
+                FileName = TestUtils.GetDotnetPath(),
+                Arguments = $"{dotnetCommandWithArgs}",
+                WorkingDirectory = this.WorkingDirectory,
+                WindowStyle = ProcessWindowStyle.Hidden,
+                RedirectStandardOutput = true,
+                RedirectStandardError = true,
+                UseShellExecute = false
+                // Note: UseShellExecute needs to be false to be able to redirect outputs and run inside workingDirectory.
+                // This also requires the full path of the dotnet process be passed to FileName.
+            };
+
+            return RunProcessWithOutputRedirect(dotnetStartInfo, out stdOutput, out stdError);
         }
 
         /// <summary>
-        /// Calls dotnet build to build the project.
+        /// Calls dotnet command on the database project.
         /// </summary>
+        /// <param name="dotnetCommand">The dotnet command to run (build, pack, etc.)</param>
         /// <param name="arguments">Any additional arguments to be passed to 'dotnet build'.</param>
         /// <returns>The Exit Code of the dotnet process.</returns>
-        /// <remarks>Adapted from Microsoft.VisualStudio.TeamSystem.Data.UnitTests.UTSqlTasks.ExecuteDotNetExe</remarks>
-        protected int Build(out string stdOutput, out string stdError, string arguments = "")
+        protected int RunDotnetCommandOnProject(string dotnetCommand, out string stdOutput, out string stdError, string arguments = "")
         {
             // Append NetCoreBuild to arguments
             arguments += " /p:NetCoreBuild=true";
```

Comment:
are we at the version of the SDK at this point to remove the netcorebuild argument?

Reply:
It should be removed already if you merge from main